src/api/v2/v2AzureResourcesApi.ts

```diff
@@ -188,6 +188,23 @@ export interface ApplicationResource extends ResourceBase {
     readonly tags?: {
         [propertyName: string]: string;
     };
+
+    /**
+     * A copy of the raw resource.
+     */
+    readonly raw: {};
+}
+
+export interface ViewPropertiesModel {
+    /**
+     * File name displayed in VS Code.
+     */
+    label: string;
+
+    /**
+     * Raw data associated with the resource to populate the properties file.
+     */
+    data: {};
```

Comment:
Maybe `Record<string, unknown>` or maybe even `unknown`?

Reply:
I do prefer `Record<string, unknown>`, but in this case it `GenericResource` doesn't satisfy `Record<string, unknown>`. `{}` is probably the next best thing, representing a non-undefined object.

Reply:
`unknown` works fine, but I have to cast it to `{}` to pass it to `openReadonlyJson`. 

<img width="946" alt="image" src="https://user-images.githubusercontent.com/12476526/201439881-f05be185-a84f-4b9c-bc84-a9f8c07f69d7.png">

Reply:
Well that's annoying. Let's just stick with `{}` then.